MemoryMap.md

```diff
@@ -27,12 +27,13 @@ be usable with DMA. However, this may well change depending on the
 DMAC that ends up being used.
 
 | Start   | Size (hex) | Size (dec) | Description                     |
-|--------:|-----------:|:----------:|:--------------------------------|
-|      $0 |   $1000    | (4KiB)     | System reserved area (note 1)   |
-|   $1000 |  $FF000    | (1020KiB)  | On-board RAM                    |
-| $100000 | $E00000    | (15MiB)    | Expansion RAM                   |
-| $F00000 |  $80000    | (512KiB)   | Addressing "hole"               |
-| $F80000 | $F00000    | (15MiB)    | Expansion RAM                   |
+|--------:|-----------:|-----------:|:--------------------------------|
+|      $0 |   $1000    | 4KiB       | System reserved area (note 1)   |
+|   $1000 |  $FF000    | 1020KiB    | On-board RAM                    |
+| $100000 | $E00000    | 15MiB      | Expansion RAM                   |
```

Comment:
Oops, I missed these changes before. 

Correct me if I'm wrong, but I think the decimal size for this line should be 14MiB, no?

Reply:
Opps! Yes, it should be 14MiB